sdk/cosmos/azure-cosmos/azure/cosmos/_query_iterable.py

```diff
@@ -91,9 +91,9 @@ def _create_execution_context(self):
     def _unpack(self, block):
         if block:
             self._did_a_call_already = False
-        return self._ex_context._continuation, block
+        return None, block
 
-    def _fetch_next(self, continuation):
```

Comment:
this is where I was confused by the continuation token - it should only be passed on the constructor, usually.